.github/workflows/htm-all-lambdas-release.yml

```diff
@@ -0,0 +1,143 @@
+name: '[Multi Region] Deploy HRM All Lambdas'
+on:
+  workflow_dispatch:
+    inputs:
+      lambda_name:
+        description: 'hrm lambda name'
+        required: true
+        default: mock
+        type: choice
+        options:
+          - mock
```

Comment:
Probably not needed here?

Reply:
Good catch. Already have it fixed in my local repo, just haven't pushed yet.